validation/src/main/scala/hmda/validation/rules/lar/validity/V485.scala

```diff
@@ -0,0 +1,31 @@
+package hmda.validation.rules.lar.validity
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.dsl.{ Predicate, Result }
+import hmda.validation.rules.EditCheck
+import scala.util.Try
+
+object V485 extends EditCheck[LoanApplicationRegister] {
+
+  def apply(lar: LoanApplicationRegister): Result = {
+    val applicant = lar.applicant
+    when(applicant.coRace1 is containedIn(1 to 5)) {
+      (applicant.coRace2 is validRace) and
+        (applicant.coRace3 is validRace) and
+        (applicant.coRace4 is validRace) and
+        (applicant.coRace5 is validRace)
+    }
```

Comment:
I think `List(applicant.race2, applicant.race3, applicant.race4, applicant.race5).forall(validRace(_)) is equalTo(true)` is decently readable, FWIW.

But I think it doesn't necessarily need to be rewritten, so I'm leaving it as is (after reverting the refactor)